### PR TITLE
fixing fileTransfer sample

### PR DESCRIPTION
**Pull request by astanley**
Discovered that the fileWriter piece of xhr2.html was no longer working. The file was not being written before the invoke command was called.  Ended up refactoring the snippet of code to correct this.  Also, while I was in there, added appropriate title HTML elements and updated the references to webworks.js.

### CODE REVIEW

File: fileTransfer/app/xhr2.html [hunk 1]
```diff
@@ -116,7 +86,44 @@
 
             alert('Error: ' + msg);
         }
-        
+
+        //This function uses the HTML5 File API to write blob data to a file
+        //Official W3 Documentation: http://www.w3.org/TR/file-system-api/
+        function saveImage (blob) {
```
Comment (ejzn) [inline fileTransfer/app/xhr2.html:92]: We have a very similar function in WP if you wanted to have a look it is under the webplatform/framework/lib/utils.js file.

Comment (astanley) [inline fileTransfer/app/xhr2.html:92]: thanks for the tip Erik

**Comment by astanley** [comment]
Can I request a code review?  Looking for a thumbs up/down for these changes.

**Comment by haahmad** [comment]
Looks good to me. r+

**Comment by timwindsor** [comment]
I think another pull that I merged has blocked this one from being automatically merged. Can you bring your branch up to date?

**Comment by pelegri** [comment]
Closing for now.  Adam, most likely you can easily sync your repo with the latest changes but I'm closing to reduce the number of open pull requests.

**Pull request closed without merging**